### PR TITLE
Doc/ota 2578/logging

### PR DESCRIPTION
**Pull request by patrickvacek**
Just updating the docs with some helpful missing information.

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -191,7 +195,23 @@ RESOURCE_CPU_WEIGHT_pn-aktualizr = "50"
 
 == Development configuration
 
-There are a few settings that can be controlled in `local.conf` to simplify the development process:
+=== Logging
+
+During development, it is recommended to enable persistent systemd logging in case you run into problems. To do so, add this to your `local.conf`:
+
+....
+IMAGE_INSTALL_append += " systemd-journald-persistent"
```
Comment (lbonn) [inline README.adoc]: Note that this is enabled by default for newly configured builds: https://github.com/advancedtelematic/meta-updater/blob/7530d93ebb97b519a12b88d121320ac0bb711c46/conf/local.conf.sample.append#L26. Maybe the phrasing could mention it?

Comment (patrickvacek) [reply README.adoc]: Done. Also added a reference to historical support for the porter board.

File: README.adoc [hunk 2]
```diff
@@ -191,7 +197,23 @@ RESOURCE_CPU_WEIGHT_pn-aktualizr = "50"
 
 == Development configuration
 
-There are a few settings that can be controlled in `local.conf` to simplify the development process:
+=== Logging
+
+During development, it is recommended to enable persistent systemd logging in case you run into problems. This is enabled by default for newly configured environments (see link:conf/local.conf.sample.append[]). To enable it manually, put this to your `local.conf`:
```
Comment (merltron) [inline README.adoc]:
```suggestion
 To troubleshoot problems that you might encounter during development, we recommend that you enable persistent `systemd` logging. This setting is enabled by default for newly configured environments (see link:conf/local.conf.sample.append[]). To enable it manually, put this to your `local.conf`:
```

Comment (patrickvacek) [reply README.adoc]: Good call. I've incorporated your suggestion.

**Pull request merged**